src/Libraries/RevitNodes/Elements/Views/ScheduleView.cs

```diff
@@ -0,0 +1,256 @@
+        //public ScheduleView RemoveField(Revit.Schedule.ScheduleField field)
+        //{
+        //    if (this.Fields.Any(x => x.Name == field.Name))
+        //    {
+        //        TransactionManager.Instance.EnsureInTransaction(DocumentManager.Instance.CurrentDBDocument);
+        //        this.InternalViewSchedule.Definition.RemoveField(field.InternalScheduleField.FieldId);
+        //        TransactionManager.Instance.TransactionTaskDone();
+        //    }
+
+        //    return this;
+        //}
+
+        /// <summary>
+        ///     Add Field.
+        /// </summary>
+        /// <param name="field"></param>
+        /// <returns></returns>
+        public ScheduleView AddField(Revit.Schedule.SchedulableField field)
+        {
+            if (!this.Fields.Any(x => x.Name == field.Name))
+            {
+                TransactionManager.Instance.EnsureInTransaction(DocumentManager.Instance.CurrentDBDocument);
+                this.InternalViewSchedule.Definition.AddField(field.InternalSchedulableField);
+                TransactionManager.Instance.TransactionTaskDone();
+            }
+
+            return this;
+        }
+
+        /// <summary>
+        ///     Schedule Fields.
+        /// </summary>
+        public List<Revit.Schedule.ScheduleField> Fields
+        {
+            get
+            {
+                IList<Autodesk.Revit.DB.ScheduleFieldId> fieldIds = this.InternalViewSchedule.Definition.GetFieldOrder();
+                List<Revit.Schedule.ScheduleField> fields = new List<Schedule.ScheduleField>();
+                if (fieldIds.Count > 0)
+                {
+                    foreach (Autodesk.Revit.DB.ScheduleFieldId id in fieldIds)
+                    {
+                        fields.Add(new Revit.Schedule.ScheduleField(this.InternalViewSchedule.Definition.GetField(id)));
+                    }
+                }
+                return fields;
+            }
+        }
+
+        /// <summary>
+        ///     Schedulable Fields.
+        /// </summary>
+        public List<Revit.Schedule.SchedulableField> SchedulableFields
+        {
+            get
+            {
+                IList<Autodesk.Revit.DB.SchedulableField> fields = this.InternalViewSchedule.Definition.GetSchedulableFields();
+                List<Revit.Schedule.SchedulableField> fields1 = new List<Schedule.SchedulableField>();
+                if (fields.Count > 0)
+                {
+                    foreach (Autodesk.Revit.DB.SchedulableField sf in fields)
+                    {
+                        fields1.Add(new Revit.Schedule.SchedulableField(sf));
+                    }
+                }
+
+                return fields1;
+            }
+        }
+
+        #endregion
+
+        #region Helpers
+
+        /// <summary>
+        /// ScheduleType Enumeration
+        /// </summary>
+        [IsVisibleInDynamoLibrary(false)]
+        public enum ScheduleType
+        {
+            KeySchedule
```

Comment:
An enum with one thing in it? Are we doing this because we think there will be more in the future?